geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/planning/QueryRunner.scala

```diff
@@ -40,6 +56,9 @@ trait QueryRunner {
 
     val query = new Query(original) // note: this ends up sharing a hints object between the two queries
 
+    // query rewriting
+    interceptors(sft).foreach(_.rewrite(query))
```

Comment:
Do we indicate if a query has been re-written?  I could imagine it being VERY helpful to list the re-writers used.  Potentially logging the query after each rewrite...  (Not sold on that, just spitballing.)

Reply:
the original query gets logged, and the query that gets executed gets logged, so you can see the delta by comparing them. There isn't an explicit message on whether it changed or not. The query interceptors are defined in the sft user data, so i don't think we need to log them each time...

Reply:
oh you mean log the query, i misunderstood

Reply:
could leave that logging up to the interceptors... i could see it both ways, but i feel like it might be kind of verbose to log the query each time, especially if it doesn't actually change - so we'd want to check for that too?

Reply:
Yeah, I know I'm thinking of something verbose...